source/cli.ts

```diff
@@ -54,14 +55,16 @@ const cli = meow(`
 		const diagnostics = await tsd(options);
 
 		if (diagnostics.length > 0) {
+			success = true;
 			throw new Error(formatter(diagnostics, showDiff));
 		}
```

Comment:
Does this need to throw an error? Maybe the "success" path should log and exit itself, and the catch block should only be for logging when `tsd` fails/crashes.

Reply:
Yeah, I don't think this should throw.